hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -165,7 +167,7 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
         }
       }
 
-      let pathLabels, values, percentage, sizeStr, pieces, subpathName;
+      let pathLabels, values: any[] = [], percentage, sizeStr, pieces, subpathName;
```

Comment:
Since values is an array of numbers, can we change to 
```suggestion
      let pathLabels, values: number[] = [], percentage, sizeStr, pieces, subpathName;
```
Let's avoid using `any` as a type as much as possible.

Reply:
@devabhishekpal 
Done changes suggested by you.